<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConsensusOptionsDialog</class>
 <widget class="QDialog" name="ConsensusOptionsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>184</width>
    <height>229</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Consensus Options</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Thresholds:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QSpinBox" name="thresholdSpinBox1_">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QCheckBox" name="enabledCheckbox1_">
       <property name="text">
        <string>Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QSpinBox" name="thresholdSpinBox2_">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>60</number>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="enabledCheckbox2_">
       <property name="text">
        <string>Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QSpinBox" name="thresholdSpinBox3_">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>70</number>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="enabledCheckbox3_">
       <property name="text">
        <string>Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QSpinBox" name="thresholdSpinBox4_">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>80</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QCheckBox" name="enabledCheckbox4_">
       <property name="text">
        <string>Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QSpinBox" name="thresholdSpinBox5_">
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>50</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>90</number>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QCheckBox" name="enabledCheckbox5_">
       <property name="text">
        <string>Enabled</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConsensusOptionsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConsensusOptionsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>207</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>228</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledCheckbox1_</sender>
   <signal>toggled(bool)</signal>
   <receiver>thresholdSpinBox1_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>128</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledCheckbox2_</sender>
   <signal>toggled(bool)</signal>
   <receiver>thresholdSpinBox2_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledCheckbox3_</sender>
   <signal>toggled(bool)</signal>
   <receiver>thresholdSpinBox3_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>120</x>
     <y>112</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledCheckbox4_</sender>
   <signal>toggled(bool)</signal>
   <receiver>thresholdSpinBox4_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>enabledCheckbox5_</sender>
   <signal>toggled(bool)</signal>
   <receiver>thresholdSpinBox5_</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>123</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>56</x>
     <y>167</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
